benchopt/benchmark.py

```diff
@@ -197,41 +198,18 @@ def _list_benchmark_classes(self, base_class):
                 # skip template solvers and datasets
                 continue
             # Get the class
-            try:
-                cls = _load_class_from_module(
-                    module_filename, class_name,
-                    benchmark_dir=self.benchmark_dir
-                )
-                if not issubclass(cls, base_class):
+            cls = _load_class_from_module(
+                module_filename, class_name,
+                benchmark_dir=self.benchmark_dir
+            )
+            if (not issubclass(cls, base_class) and
+                cls.__name__ != "FailedImport"):
```

Comment:
@tomMoral this is to avoid importing the FailedImport class, as it would required defining it outside of the function it's used in